web-client/src/presenter/computeds/pendingReportListHelper.ts

```diff
@@ -16,9 +16,9 @@ export const pendingReportListHelper = (
 
   const showLoadMore =
     get(state.pendingReports.pendingItems).length < searchResultsCount;
-
-  const showNoPendingItems = searchResultsCount === 0 && hasPendingItemsResults;
   const showSelectJudgeText = !judge;
+  const showNoPendingItems =
+    searchResultsCount === 0 && !hasPendingItemsResults && !!judge;
 
```

Comment:
I think after the updates, this variable is named correctly. We want to hide/show elements based on whether there actually _are_ pending item results, rather than whether the report has been run or not (this is where the bug came from)